### PR TITLE
Add docs for formula_one integration

### PR DESCRIPTION
**Pull request by y34hbuddy**
## Proposed change

New core integration: formula_one

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [X] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/80326
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/3786
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/formula_one.markdown [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ha_domain: formula_one
+ha_platforms:
+  - sensor
+ha_codeowners:
+  - '@y34hbuddy'
+ha_integration_type: integration
+ha_config_flow: true
+---
+
+The `formula_one` integration offers integration with the [Ergast](http://ergast.com/mrd/) Formula 1 API to provide details about the current season's Drivers Championship, Constructors Championship, and Race schedule.
```
Comment (frenck) [inline source/_integrations/formula_one.markdown]:
```suggestion
The Formula 1 integration offers integration with the [Ergast](http://ergast.com/mrd/) Formula 1 API to provide details about the current season's Drivers Championship, Constructors Championship, and Race schedule.
```

File: source/_integrations/formula_one.markdown [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+### Available Resources
+
+The list of currently available resources:
+
+- `f1_driver_##`: sensors representing each position in the Drivers Championship
+- `f1_driver_DRIVER_NAME`: sensors representing each Driver in the Drivers Championship
+- `f1_constructor_##`: sensors representing each position in the Constructors Championship
+- `f1_constructor_CONSTRUCTOR_NAME`: sensors representing each Constructor in the Constructors Championship
+- `f1_race_##`: sensors representing each Race of the season
+- `f1_next_race`: a sensor representing the next Race of the season
```
Comment (frenck) [inline source/_integrations/formula_one.markdown]: What are these codes in those code blocks? Entity IDs? Or?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/formula_one.markdown (hunk 2 above)
Comment (y34hbuddy) [reply source/_integrations/formula_one.markdown]: Indeed - these are sensor.* entity IDs. Is there a more-preferred way of documenting these?

Comment (frenck) [reply source/_integrations/formula_one.markdown]: Aah ok, it wasn't really clear those where entity IDs (as those start with e.g., `sensor.`). Anyways, it is not preferred to write entity IDs, as those are not guaranteed to be assigned.

Instead, just describe them in human language.

Comment (y34hbuddy) [reply source/_integrations/formula_one.markdown]: Understood - will do!

File: source/_integrations/formula_one.markdown [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+ha_domain: formula_one
+ha_platforms:
+  - sensor
+ha_codeowners:
+  - '@y34hbuddy'
+ha_integration_type: integration
+ha_config_flow: true
+---
+
+The Formula 1 integration offers integration with the [Ergast](http://ergast.com/mrd/) Formula 1 API to provide details about the current season's Drivers Championship, Constructors Championship, and Race schedule.
```
Comment (hmmbob) [inline source/_integrations/formula_one.markdown]:
```suggestion
The Formula 1 integration offers integration with the [Ergast](https://ergast.com/mrd/) Formula 1 API to provide details about the current season's Drivers Championship, Constructors Championship, and Race schedule.
```

**Comment by y34hbuddy** [comment]
Apologies for re-requesting review... that was meant for another PR  :)

File: source/_integrations/formula_one.markdown [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  - '@y34hbuddy'
+ha_integration_type: integration
+ha_config_flow: true
+---
+
+The Formula 1 integration offers integration with the [Ergast](https://ergast.com/mrd/) Formula 1 API to provide details about the current season's Drivers Championship, Constructors Championship, and Race schedule.
+
+{% include integrations/config_flow.md %}
+
+### Available Resources
```
Comment (frenck) [inline source/_integrations/formula_one.markdown]:
```suggestion
## Available resources
```

File: source/_integrations/formula_one.markdown [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+The list of currently available resources:
+
+- Sensors representing each position in the Drivers Championship
+- Sensors representing each Driver in the Drivers Championship
+- Sensors representing each position in the Constructors Championship
+- Sensors representing each Constructor in the Constructors Championship
+- Sensors representing each Race of the season
+- A sensor representing the next Race of the season
```
Comment (frenck) [inline source/_integrations/formula_one.markdown]:
```suggestion
The list of currently available resources:

- Sensors representing each position in the Drivers Championship
- Sensors representing each driver in the Drivers Championship
- Sensors representing each position in the Constructors Championship
- Sensors representing each constructor in the Constructors Championship
- Sensors representing each race of the season
- Sensor representing the next race of the season
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**